cmake/functions.cmake

```diff
@@ -32,9 +32,6 @@ function(addtest_part test_name)
   target_sources(${test_name} PUBLIC
       ${ARGN}
       )
-  target_link_libraries(${test_name}
```

Comment:
?

Reply:
Linker complained about "duplicate" library.
`GTest::gmock_main` (above) already links to `GTest::gtest`.